Source/WebCore/html/HTMLInputElement.cpp

```diff
@@ -2154,10 +2154,11 @@ static Ref<StyleGradientImage> autoFillStrongPasswordMaskImage()
 
     return StyleGradientImage::create(
         StyleGradientImage::LinearData {
-            nullptr, nullptr, nullptr, nullptr, CSSValuePool::singleton().createValue(90, CSSUnitType::CSS_DEG)
+            {
+                CSSLinearGradientValue::Angle { CSSValuePool::singleton().createValue(90, CSSUnitType::CSS_DEG) }
+            },
+            CSSGradientRepeat::NonRepeating
```

Comment:
I never knew the `input` element implementation had gradients, but now I do I remember exactly where it is. And that UI sort of annoys me :)